voronoi_game.py

```diff
@@ -32,6 +31,11 @@ def __init__(self, num_stones, num_players, grid_size, min_dist, host, port, use
       self.graphic_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
       self.graphic_socket.connect(('localhost', 8080))
 
+    self.row_numbers = np.zeros((grid_size, grid_size), dtype=np.float32)
```

Comment:
What are these lines here? You never use them again right?

Reply:
These are used for pull-calculations.